jaeger_client/thrift.py

```diff
@@ -82,6 +83,12 @@ def make_tag(key, value, max_length):
             key=key,
             value=value
         )
+    elif type(value).__name__ == 'traceback':
```

Comment:
is every possible value guaranteed to have `type(value).__name__` expression working?

Reply:
I'm not completely sure it's guaranteed, but line 71 uses the same expression to check for booleans, so the newly added line shouldn't cause any new issues.

Reply:
+1